translation/source/emails.xml

```diff
@@ -13,9 +13,9 @@
   <string name="emailChange_click">To confirm you have access to this email, please click the link below:</string>
 
   <string name="welcome_subject">Welcome to lichess.org, %s</string>
-  <string name="welcome_text">You have successfully created your account on https://lichess.org.
+  <string name="welcome_text">You have successfully created your account on %1$s.
```

Comment:
Some translations already exist and would be broken by adding a new placeholder. It's probably best to change the name of this string.